doc/uftrace-report.md

```diff
@@ -24,25 +24,34 @@ REPORT OPTIONS
 ==============
 -f *FIELD*, \--output-fields=*FIELD*
 :   Customize field in the output.  Possible values are: `total`, `total-avg`,
-    `total-min`, `total-max`, `self`, `self-avg`, `self-min`, `self-max`, `size`,
-    `call` and `all`.  Multiple fields can be set by using comma.  Special field
-    of 'none' can be used (solely) to hide all fields and 'all' can be used to
-    show all fields.
-    Default is 'total,self,call'.  See *FIELDS*.
+    `total-min`, `total-max`, `total-stdv`, `self`, `self-avg`, `self-min`, `self-max`,
+    `self-stdv`, `size`, `call` and `all`.  Multiple fields can be set by using comma.
+    Special field of 'none' can be used (solely) to hide all fields and 'all' can be used
+    to show all fields.
+    Default is 'total,self,call'. See *FIELDS*.
+    `stdv` stands for the relative standard deviation (RSD),
+    which is the standard deviation expressed as a percentage of the mean.
+    The relative standard deviation is calculated using the formula:
+
+    \[ RSD = \left( \frac{\sigma}{\mu} \right) \times 100\% \]
+
+    where:
+    - \( \sigma \) represents the standard deviation.
+    - \( \mu \) represents the mean of the function's time.
```

Comment:
I'm leaving a comment very late but this part isn't shown properly in its markdown format.
https://github.com/namhyung/uftrace/blob/master/doc/uftrace-report.md#report-options

It is shown as follows in its man page.
```
              [ RSD = ( ) % ]

              where: - ( ) represents the standard deviation.  - ( ) represents the mean of the function’s time.
```
Can we fix it?